capture_service/android_application.cc

```diff
@@ -137,8 +137,10 @@ absl::Status AndroidApplication::ParsePackage()
 absl::Status AndroidApplication::Start()
 {
     RETURN_IF_ERROR(m_dev.Adb().Run("shell input keyevent KEYCODE_WAKEUP"));
-    RETURN_IF_ERROR(m_dev.Adb().Run(
-    absl::StrFormat("shell am start -S -W %s -n %s/%s ", m_command_args, m_package, m_main_activity)));
+    RETURN_IF_ERROR(m_dev.Adb().Run(absl::StrFormat("shell am start -S -W %s -n %s/%s ",
```

Comment:
Looks like there's some whitespace that can be trimmed from the command:
```suggestion
    RETURN_IF_ERROR(m_dev.Adb().Run(absl::StrFormat("shell am start -S -W %s -n %s/%s",
```